docs/source/guide/tutorials/advanced/semi_sl.rst

```diff
@@ -2,6 +2,141 @@
 Use Semi-Supervised Learning
 ############################
 
-<Guidelines for writing>
+This tutorial provides instructions on how to use semi-supervised learning with OTX.
 
-This tutorial explains how to employ semi-supervised learning with OTX.
+OTX now offers semi-supervised learning, which combines labeled and unlabeled data during training to improve model accuracy. Currently, this type of training is available for multi-class classification, object detection, and semantic segmentation.
+
+Semi-supervised learning will soon be available for multi-label classification and instance segmentation as well.
+
+If want to learn more about the algorithms used in semi-supervised learning, please refer to the explanation section below.
+
+- :doc:`Multi-class Classification <../../explanation/algorithms/classification/multi_class_classification>`
```

Comment:
Please, add there direct link to Semi-SL sections (via # or direct reference as I did in classification sections)